tests/unit/cartography/intel/okta/test_admin_roles.py

```diff
@@ -0,0 +1,32 @@
+from cartography.intel.okta.oktaintel import transform_group_roles_data
+from cartography.intel.okta.oktaintel import transform_user_roles_data
+from tests.data.okta.adminroles import LIST_ASSIGNED_GROUP_ROLE_RESPONSE
+from tests.data.okta.adminroles import LIST_ASSIGNED_USER_ROLE_RESPONSE
+
+
+def test_transform_user_roles():
+    org_id = "lyft"
+    props_list = transform_user_roles_data(LIST_ASSIGNED_USER_ROLE_RESPONSE, org_id)
+
+    assert len(props_list) == 2
```

Comment:
Rather than using 6 separate assertions, you can use a single assert to compare complex data types. E.g. `assert actual == [{'foo': 'bar'}, 1, 2, 3]` and pytest will figure out where mismatches exist and fail the assertion accordingly. This works reliably with hashable data types, so all the basic Python data types will work great. I recommend replacing the lengthy lists of asserts through these test modules with structured comparisons like this.

Reply:
Not going to comment on every assert, but the above is feedback for all test files.